src/components/Lookup/__test__/selectedValue.spec.js

```diff
@@ -27,10 +27,24 @@ describe('<SelectedValue />', () => {
         const component = mount(<SelectedValue />);
         expect(component.find('ButtonIcon').exists()).toBe(true);
     });
-    it('should fire an event when click the button', () => {
+    it('should fire an event when the close button is clicked', () => {
         const onClearMockFn = jest.fn();
         const component = mount(<SelectedValue onClearValue={onClearMockFn} />);
         component.find('ButtonIcon').simulate('click');
         expect(onClearMockFn).toHaveBeenCalledTimes(1);
     });
+    it('should set the right class names when readOnly is passed', () => {
+        const component = mount(<SelectedValue readOnly />);
+        expect(component.find('div.rainbow-lookup_selected-value--readonly').exists()).toEqual(
+            true,
+        );
+    });
+    it('should not render the close button when readOnly is passed', () => {
+        const component = mount(<SelectedValue readOnly />);
+        expect(component.find('ButtonIcon').exists()).toEqual(false);
+    });
+    it('should not render the close button when disabled is passed', () => {
+        const component = mount(<SelectedValue disabled />);
+        expect(component.find('ButtonIcon').exists()).toEqual(false);
```

Comment:
use toBe instead of toEqual
toEqual is for array and objects equality check
in order to compare strings, numbers and booleans use toBe